src/components/c-modal.vue

```diff
@@ -6,53 +6,43 @@
     <dialog v-if="isOpen"
             :class="b(modifiers)"
     >
-      <div ref="container" :class="b('container')">
-        <div v-outside-click="onOutsideClick"
-             ref="inner"
-             :class="b('inner')"
-        >
-          <div v-if="$slots.head || title || isClosable" :class="b('header')">
-            <div :class="b('header-inner')">
-              <slot name="head" :close="close">
-                <h2 v-if="title" :class="b('title')">
-                  {{ title }}
-                </h2>
-                <button v-if="isClosable"
-                        :aria-title="$t('c-modal.buttonClose')"
-                        :class="b('button-close')"
-                        type="button"
-                        @click="close"
-                >
-                  <e-icon icon="i-close" size="30" />
-                </button>
-              </slot>
-            </div>
-          </div>
-          <div :class="b('content')">
-            <slot></slot>
-          </div>
-          <div v-if="$slots.stickyFooter"
-               ref="stickyFooter"
-               :class="b('sticky-footer')"
-          >
-            <slot name="stickyFooter"></slot>
+      <div v-outside-click="onOutsideClick" :class="b('inner')">
+        <div v-if="$slots.head || title || isClosable" :class="b('header')">
+          <div :class="b('header-inner')">
+            <slot name="head" :close="close">
+              <h2 v-if="title" :class="b('title')">
+                {{ title }}
+              </h2>
+              <button v-if="isClosable"
+                      :aria-title="$t('c-modal.buttonClose')"
```

Comment:
todo: `aria-title` is invalid. You should use `aria-label` (and `title` if you want to display a tooltip for mouse).